android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -314,7 +314,7 @@ private void setupPurchases(String apiKey, String appUserID, @Nullable Boolean o
             result.success(null);
         } else {
             result.error(
-                    String.valueOf(PurchasesErrorCode.UnknownError.ordinal()),
+                    String.valueOf(PurchasesErrorCode.UnknownError.code),
```

Comment:
This particular usage worked correctly because it was index `0`, but it's still a good idea to update.

Reply:
Thanks for checking for usages

Reply:
just tested, the method is actually supposed to be `getCode()` since it gets synthetized for java. updated the usage and tested.